### PR TITLE
Add a code of conduct

### PR DESCRIPTION
**Pull request by dabacon**
Version per recommendation of Google's open source team.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+This Code of Conduct also applies outside the project spaces when the Project
+Steward has a reasonable belief that an individual's behavior may have a
+negative impact on the project or its community.
+
+## Conflict Resolution
+
+We do not believe that all conflict is bad; healthy debate and disagreement
+often yield positive results. However, it is never okay to be disrespectful or
+to engage in behavior that violates the project’s code of conduct.
```
Comment (viathor) [inline CODE_OF_CONDUCT.md:63]: nit: Perhaps capitalize "Code of Conduct" for consistency with the rest of the text.

(Applies also to l.65 and l.74 below.)

Comment (dabacon) [reply CODE_OF_CONDUCT.md]: done

**Comment by dabacon** [comment]
@googlebot the cla is ok

**Comment by viathor** [comment]
Googlebot is a Toydarian. Jedi tricks don't work on them. Only money.

**Comment by dabacon** [comment]
@googlebot rescan

**Comment by dabacon** [comment]
Thank you googlebot, your, ahem, gift is in the mail.

**Pull request merged**